wordpress-mobile/WordPress-iOS
# Multi wp account support

Fixes #6204 

## Comment
As discussed privately, I was supposed to use 'userID' to identify uniquely an account, but that is set in a later in time, so I used 'manageObjectID'


To test:

Log In with an WP account and see the 'Add Account' option, pressing it will show the log in screen letting the user add another wp account. 
On 'Me' section tapping on the navigation bar should show a list of current logged in users, selecting an account from that list should refresh all the sections in the app

## Next Steps

- [ ] MeViewController needs a refactor so the dependencies are injected and more tests can be added _needs issue_
- [ ] Fix the Blog icon bug explained below _needs issue_
- [ ] Fix Notifications refreshing issue _needs issue_
- [ ] Keep the credentials so in welcome screen we can show options to the user for accounts already used in the app (we will only keep the credentials) _needs issue_  

## Known Issues

- There is a crash which I couldn't fin the steps for, where 'sometimes'  '- (void)setBlog:(Blog *)blog' will crash because although there is a blog there is no blog.icon.
- Notifications reacts to accountDidChange notification, and triggers a refresh process, but it doesn't really update the UI.

## Demo




Needs review: @jleandroperez @aerych
